### ISSUE TITLE
50 moves

### ISSUE THREAD

**Issue by Krzychumag** [cutechess/cutechess#361]
Please add a new option "50".The option means that after 50 moves without whipping a pawn or figure is a draw.

**Comment by ilaripih** [comment]
The FIDE 50 move rule is already enforced. Do you mean something different?

**Comment by Krzychumag** [comment]
Yes. Imagine this situation. They play two chess engines which in their options have no option that after 50 moves is a draw. The effect will be that after 50 movements, the engines will continue to play.

**Comment by ilaripih** [comment]
It doesn't matter if the engines don't know the 50 move rule; cutechess will adjudicate the game as a draw because it enforces the 50 move rule. Or do you have a game (with PGN output) where cutechess fails to adjudicate the draw?
